core/vmpu/src/kinetis/vmpu_kinetis_mem.c

```diff
@@ -63,20 +63,30 @@ void vmpu_mem_switch(uint8_t src_box, uint8_t dst_box)
 
 void vmpu_mem_init(void)
 {
+    /* UART DMA is bus master 2 (configured through M2UM). ENET DMA is bus
+     * master 3 (configured through M3UM). */
+    static const uint32_t bmum_read_bit = (1 << 2);
+    static const uint32_t bmum_write_bit = (1 << 1);
+#define M3UM 18
```

Comment:
@Patater The mixture of #define and static const is not acceptable. Either both need to be #defines (namespaced and at the top of the file) or both static consts. ALso the abbreviated name M3UM is not useful.

Alternatively please use a single RW define/const (feel free to add a RO and WO define, too) for readability.

My preference though is to keep the hardcoded hex number, in combination with the comment above - unless you are ready to add proper macros and all defines for this register. It's more readable when verifying that against the K64F spec.